waiter/src/waiter/client_tools.clj

```diff
@@ -240,7 +249,7 @@
          (when verbose
            (log/info (get request-headers "x-cid") "response size:" (count (str response-body))))
          {:request-headers request-headers
-          :cookies (cookies/get-cookies (.getCookieStore client))
+          :cookies (parse-cookies (get headers "set-cookie"))
```

Comment:
Is `[qbits.jet.client.cookies :as cookies]` an unused import and hence should be removed?

Reply:
Done